README.md

```diff
@@ -154,6 +156,8 @@ These are the years-of-introduction of various frameworks and library collection
 
 - Symfony 3.0.0: <http://symfony.com/blog/symfony-3-0-0-released>
 
+- Simple MVC Framework 2.1: <https://packagist.org/packages/simple-mvc-framework/v2#v2.1>
```

Comment:
As you may have noticed there is no minor versions like 2.1.1 , 3.6.1, etc. Only major releases 1.0.0, 2.0.0, 3.0.0, etc